_opensearch/segment-replication/index.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: default
+title: Segment Replication Feature
+nav_order: 122
+parent: Opensearch
+has_children: true
+redirect_from:
+  - /opensearch/segment-replication/
+---
+
+## Segment Replication
+
+Opensearch 2.3 contains an experimental version of segment replication.
+
+With segment replication, segment files are copied across shards instead of indexing documents on each shard copy. This improves indexing throughput at the expense of increased network utilization.
+
```

Comment:
Can we elaborate on why the users would want to enable this feature?

1. Specific users who have very high write requirements but not a high search requirements  will benefit from...
2. This is the first step in a series of features to decouple reads and writes (the cost for adding replicas goes down from a compute standpoint). So, you can run on a cheaper hardware (add more replicas without impacting your write performance).

Reply:
added back uses cases and changed opening wording